onnx/helper.py

```diff
@@ -16,6 +16,57 @@
 from typing import Text, Sequence, Any, Optional, Dict, Union, TypeVar, Callable, Tuple, List, cast
 import numpy as np  # type: ignore
 
+VersionRowType = Union[Tuple[Text, int, int, int], Tuple[Text, int, int, int, int]]
+VersionTableType = List[VersionRowType]
+
+# This is a copy of the documented version in https://github.com/onnx/onnx/blob/master/docs/Versioning.md#released-versions
+# Both must be updated whenever a new version of ONNX is released.
```

Comment:
This table needs to be updated before every release right? Can you change this to something like : This must be updated before creating release candidate packages... 
Exactly when the doc is updated does not matter as much as when this table is updated.

Reply:
I am not sure I understand the distinction you want to make. It seems important that both are updated. It seems better to update both at the same time (ideally). As to the "when" question: whenever we start a new opset version (for ai.onnx or ai.onnx.ml etc.), it seems to me we should update all of these ... does that sound right?

Reply:
I got confused by the wording:
Both must be updated whenever a new version of ONNX is **released** 
This made me think that the updates need to happen before a release and that is why I made the comment that it should be clear that the update needs to happen before the release package is created (with documentation 1 can always update them after the release)

What you said now in the comment makes more sense to me - 
whenever we start a new opset version (for ai.onnx or ai.onnx.ml etc.), it seems to me we should update all of these